src/main/java/rocks/inspectit/gepard/agent/transformation/DynamicTransformer.java

```diff
@@ -51,7 +52,9 @@ public DynamicType.Builder<?> transform(
       JavaModule module,
       ProtectionDomain protectionDomain) {
     InstrumentedType currentType = new InstrumentedType(typeDescription.getName(), classLoader);
-    if (resolver.shouldInstrument(typeDescription)) {
+    ClassInstrumentationConfiguration currentConfig =
```

Comment:
It would also make sense to ask for the state at this point, instead of resolving something.
state.getClassInstConfigByType(type) or something